various updates (#9503)
MicrosoftDocs/office-docs-powershell: skype/skype-ps/skype/Grant-CsCallingLineIdentity.md

title: Grant-CsCallingLineIdentity
schema: 2.0.0
manager: bulenteg
author: jenstrier
ms.author: jenstr
ms.reviewer:
---

# Grant-CsCallingLineIdentity

## SYNOPSIS
Use the `Grant-CsCallingLineIdentity` cmdlet to apply a Caller ID policy to a user account or set the tenant Global instance.

## SYNTAX

### GrantToTenant (Default)
```
Grant-CsCallingLineIdentity [[-PolicyName] <string>] [-Global] [-PassThru] [-WhatIf] [-Confirm] [<CommonParameters>]
```
### Identity
```
Grant-CsCallingLineIdentity [[-Identity] <string>] [[-PolicyName] <string>] [-PassThru] [-WhatIf] [-Confirm] [<CommonParameters>]
```

## DESCRIPTION

> You can either a Caller ID policy to a specific user or you can set the Global policy instance.

Reviewer: ```suggestion
You can either create a Caller ID policy for a specific user or you can set the Global policy instance.
```

Reviewer: @JensTrier, I am not sure about this. Please feel free to change it. However, the original line is incomplete.

Accept wildcard characters: False
```

### -PolicyName
The name (Identity) of the Caller ID policy to be assigned to the user. To remove an existing user level policy assignment, specify PolicyName as $null.

```yaml
Type: String
Parameter Sets: (All)
Aliases: 
Applicable: Microsoft Teams, Skype for Business Online

Required: False
Position: 1
Default value: None
Accept pipeline input: False
Accept wildcard characters: False
```

### -Identity

> The Identity of the user to whom the policy is being assigned. User Identities can be specified using the user's SIP address, the user's user principal name (UPN) or,

Reviewer: ```suggestion
The Identity of the user to whom the policy is being assigned. User Identities can be specified using the user's SIP address, the user's user principal name (UPN), or
```

Accept wildcard characters: False
```

### -Confirm

> The Confirm switch causes the command to pause processing, and requires confirmation to proceed.

Reviewer: ```suggestion
The Confirm switch causes the command to pause processing and requires confirmation to proceed.
```
